lib/src/util/is_valid_address.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:solana/src/base58/base58.dart';
+import 'package:solana/src/edwards25519/edwards_25519.dart';
+
+bool isValidAddress(String address) {
```

Comment:
Since it's exported function, there should be some documentation on it.

Reply:
True! It's just that when I finished it was late and I didn't notice. Will add it! Thanks!